src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0, // result is a property of the calculator object
+
+    // Define the basic operations
+    add: function (num) {
+      this.result += num;
+    },
+    subtract: function (num) {
+      this.result -= num;
+    },
+    multiply: function (num) {
+      this.result *= num;
+    },
+    divide: function (num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        return null;
+      }
+    },
```

Comment:
[CODE KNOWLEDGE] The same applies for the `divide` method. Use the shorter syntax `divide(num) {...}` instead of `divide: function (num) {...}`.